src/app/services/tailwind-theme.service.ts

```diff
@@ -0,0 +1,104 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+export interface CpeColorTheme {
+    black: string;
+    white: string;
+    primary: string;
+    secondary: string;
+    highlight: string;
+    disabledText: string;
+    disabledBackground: string;
+    selected: string;
+    checked: string;
+    unchecked: string;
+    warningText: string;
+    warningBackground: string;
+}
+
+export const defaultTheme = { // fallback theme
+    "black": "#000000",
+    "white": "#FFFFFF",
+    "primary": "#336790",
+    "secondary": "#FFFFFF",
+    "highlight": "#74A2CF",
+    "disabledText": "#9CA3AF",
+    "disabledBackground": "#E5E7EB",
+    "selected": "#D6E0E8",
+    "checked": "#D6E0E8",
+    "unchecked": "#E5E7EB",
+    "warningText": "#F59E0B",
+    "warningBackground": "#FEF3C7",
+    "surface": "#000000",
+    "surfaceText": "#FFFFFF"
+}
```

Comment:
This object with the same values is found in the _cpeTheme.json_ . It would make sense to reuse code